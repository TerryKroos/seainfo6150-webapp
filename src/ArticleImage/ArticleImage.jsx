import React from "react";
import styles from "./ArticleImage.module.css";

const ArticleImage = (props) => {
    return (
      <div className={styles.container}>
       <img src={props.url} alt={props.title} />
      </div>
    );
  };

  export default ArticleImage;