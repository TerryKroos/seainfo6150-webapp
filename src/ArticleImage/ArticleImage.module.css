.container{
    width:100px;
    margin-bottom: 10px;
}
img {
    border: 5px solid red;
    border-radius: 50%;
    width: 100%;
    height: auto;

}