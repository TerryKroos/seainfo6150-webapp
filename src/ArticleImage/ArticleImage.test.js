import React from "react";
import ArticleImage from "./ArticleImage";

describe("FunctionalComponent tests", () => {
  it("renders correctly", () => {
    const { container } = render(<ArticleImage title="" url=""/>);
    expect(container).toMatchSnapshot();
  });
})