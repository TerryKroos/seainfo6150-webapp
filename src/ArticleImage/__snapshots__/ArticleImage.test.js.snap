// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FunctionalComponent tests renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <img
      alt=""
      src=""
    />
  </div>
</div>
`;
